### DISCUSSION TITLE
Saving to default semantic model of a datamart

### DISCUSSION THREAD

**Discussion by kumaiJoel** [TabularEditor/TabularEditor3#1357] [Data Modeling]
Hi community

We have a datamart in place in Fabric for which we would like to use Tabular Editor 3 (Business) to connect and make changes to the model. Connection works fine but upon saving some hierarchies the saving fails. Even when being the owner of the datamart itself and checking the permissions of both the datamart itself and the default semantic model, we always run into a permission error: 

![image](https://github.com/user-attachments/assets/0fbf566c-3526-49d4-9d57-bd33a67da1d8)

Has anyone ever tried to to the same? is that a limitation know to default semantic models?

**Comment by otykier** [comment]
It’s unfortunately [not possible to make changes to default semantic models through the XMLA endpoint](https://learn.microsoft.com/en-us/power-bi/enterprise/service-premium-connect-tools#considerations-and-limitations).

Consider making and deploying a copy of the model, if you need to make changes using Tabular Editor.
